### DISCUSSION TITLE
How to format tags for createDraftListing

### DISCUSSION THREAD

**Discussion by krisograbek** [etsy/open-api#1040] [Q&A]
I'm successfully creating a new listing draft via API.

However, in the draft, I don't see the full list of tags I'm sending over.

I only see the last tag. I'm using a simple list in Python to send it.

For example, when I send:
`"tags": ["rainforest", "mystical", "magical", "dreamy forest"]`
I see only "dreamy forest" in the draft.

Is there any specific format of tags that API expects? In the documentation, it says "array of strings"

**Comment by emergingdzns** [comment]
Don't send as an array. Send it is a comma delimited string. No quotes. Their system will bring it up by the commas and make the tags.

**Reply by krisograbek** [reply]
I've tried that but got an error that a tag is too long (over 20 characters)

I thought it was because I was sending a long string.

I just realized that one of the tags I was sending was actually too long...

Thanks!

**Reply by emergingdzns** [reply]
Yeah they are limited to 20 characters each. Please mark my response above as the answer so that the community knows that your question has been answered. This helps others that are looking for answers to the same question.

**Comment by GorgonFreeman** [comment]
To be completely unhelpful, I am using NodeJS and have had no issues submitting arrays of tags.
